new_script.ps1

```diff
@@ -0,0 +1,172 @@
+#This script builds several branches for an Application in AppCenter and saves a report.
+#Check Readme.md file for more details - https://github.com/bumbazimba/appcenter_automatization/blob/master/README.md
+#Designed for Powershell 6 or above. Tested on Powershell 7
+#Author Nikita Nikolaev
+#Version 1.1
+
+#Enter AppCenter credentials for API requests
+
+[string]$user = Read-Host "Enter AppCenter username"
+[string]$app = Read-Host "Enter Application name"
+[string]$token = Read-Host "Enter API Token"
+[int]$branches_number = Read-Host "Enter number of branches to build"
+$branches_number = $branches_number-1
+
+
+$excel = Create-ExcelFile
+$branches = Get-Branches
+$build = Build-Branches $branches

```

Comment:
should be plural i guess as later on we assign multiple values to it.

Reply:
changed $build to $builds